pkg/gateway/server/dispatcher/dispatcher.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 type Dispatcher struct {
-	invoker                     *invoke.Invoker
```

Comment:
This should remain unexported. If you need the invoker somewhere else, then you can use the `services.Invoker`. There is nothing special about this one.